tasks_downstream.py

```diff
@@ -183,3 +183,28 @@ def logs(c, tail=10):
         cmd += f" --tail {tail}"
     with c.cd(str(PROJECT_ROOT)):
         c.run(cmd)
+
+
+@task(develop)
+def psql(c, db=None):
+    """Get a psql"""
+    cmd = "docker-compose run --rm odoo psql"
+    if db:
+        cmd += f" {db}"
+    c.run(cmd)
+
+
+@task(develop)
+def shell(c, db=None):
+    """Get an Odoo shell"""
+    cmd = "docker-compose run --rm odoo odoo shell"
```

Comment:
```suggestion
    cmd = "docker-compose run --rm odoo click-odoo"
```

This provides a shell out of the box for v8 too.